system/autoware_hazard_status_converter/src/converter.cpp

```diff
@@ -23,6 +24,19 @@ namespace autoware::hazard_status_converter
 
 Converter::Converter(const rclcpp::NodeOptions & options) : Node("converter", options)
 {
+  if (declare_parameter<bool>("include_latent_fault_in_emergency")) {
+    emergency_threshold_ = HazardStatus::LATENT_FAULT;
+  } else {
+    emergency_threshold_ = HazardStatus::SINGLE_POINT_FAULT;
+  }
+
+  use_external_emergency_holding_ = declare_parameter<bool>("use_external_emergency_holding");
+  if (use_external_emergency_holding_) {
+    sub_emergency_holding_ =
+      autoware_utils_rclcpp::InterProcessPollingSubscriber<EmergencyHolding>::create_subscription(
+        this, "~/input/emergency_holding");
+  }
```

Comment:
Looking [these lines](https://github.com/autowarefoundation/autoware_universe/pull/11721/files#diff-6786e819257c3cc22d4d79b2106364a5fd3d29174283b147b2ca7d64700fe261R142-R145) too, it seems the `sub_emergency_holding_` is used only when `use_external_emergency_holding_ = true`.

Perhaps using `std::optional<...>` for `sub_emergency_holding_` might be one of the answers (then perform `take_data()` if `sub_emergency_holding_.has_value() == true`). But personally it is okay with the current implementation :+1: